modules/apps/portal-search/portal-search-api/src/main/java/com/liferay/portal/search/configuration/ReindexConfiguration.java

```diff
@@ -23,7 +23,7 @@
 public interface ReindexConfiguration {
 
 	@Meta.AD(
-		deflt = "regular", description = "default-reindex-execution-mode-help",
+		deflt = "full", description = "default-reindex-execution-mode-help",
```

Comment:
i think this kind of change typically causes a micro semver bump, did you run baseline for portal-search-api?

Reply:
Thanks, added a commit to up the version! Let me know if it's not alright 👍